pyfr/integrators/dual/controllers.py

```diff
@@ -50,8 +52,8 @@ def __init__(self, *args, **kwargs):
             raise ValueError('The maximum number of pseudo-iterations must '
                              'be greater than or equal to the minimum')
 
-        self._pseudo_aresid = self.cfg.getfloat(sect, 'pseudo-aresid')
-        self._pseudo_rresid = self.cfg.getfloat(sect, 'pseudo-rresid')
+        self._residtol= self.cfg.getfloat(sect, 'pseudo-residtol')
+        self._norm = self.cfg.get(sect, 'resid-norm', 'uniform')
```

Comment:
Do you mean by removing the default norm? Yes, defaults make user think/experiment less...